src/config.rs

```diff
@@ -17,17 +17,19 @@
 use std::collections::HashMap;
 use std::env;
 use std::fmt;
-use std::fs;
 
 use log::error;
 use serde_derive::{Deserialize, Serialize};
 
-use crate::constants::DEFAULT_SERVER_ADDRESS;
+use crate::constants::{
+    DEFAULT_SERVER_ADDRESS, METABUCKET_ACCESS_KEY, METABUCKET_ENDPOINT, METABUCKET_NAME,
+    METABUCKET_SECRET_KEY, PKCS12_CERT, PKCS12_PASSWORD,
```

Comment:
Environment variable names are being used to configure the server, so they are not needed in other modules. I don't see why they need to be in a separate module and imported here like this. It is simpler to just put the constants in the config module.